io-engine/src/core/env.rs

```diff
@@ -163,9 +196,15 @@ pub struct MayastorCliArgs {
     #[clap(short = 'T', long = "tgt-iface", env = "NVMF_TGT_IFACE")]
     /// NVMF target interface (ip, mac, name or subnet).
     pub nvmf_tgt_interface: Option<String>,
-    /// NVMF target Command Retry Delay.
-    #[clap(long = "tgt-crdt", env = "NVMF_TGT_CRDT", default_value = "0")]
-    pub nvmf_tgt_crdt: u16,
+    /// NVMF target Command Retry Delay in x100 ms (single integer or three
+    /// comma-separated integers).
+    #[clap(
+        long = "tgt-crdt",
+        env = "NVMF_TGT_CRDT",
+        default_value = "0",
+        value_parser = parse_crdt,
+    )]
+    pub nvmf_tgt_crdt: [u16; 3],
```

Comment:
So, if I understand this correctly, the first 2 are for the nexus and the last one for the replicas?
Would be good to document this here as well, including when they are used.
Also, could we have completely separate  crdt array for nexus and replica?

Reply:
So it is configured differently for two targets? Not at the moment, as we have one target, listening on two ports. CRD is configured per target, not per port.

Reply:
Add documentation comments as well.